lib/scenes/main_menu.cpp

```diff
@@ -12,47 +12,75 @@
 #include "brickengine/rendering/renderables/data/color.hpp"
 #include "brickengine/json/json.hpp"
 #include "controllers/game_controller.hpp"
+#include "brickengine/components/player_component.hpp"
 
 MainMenu::MainMenu(EntityFactory& factory, BrickEngine& engine, GameController& game_controller)
     : Menu(factory, engine, WIDTH, HEIGHT), game_controller(game_controller) { }
 
 void MainMenu::performPrepare() {
     // General information
-    this->bg_path = "colors/white.png";
+    this->bg_path = "backgrounds/pixel-forest.png";
     this->bg_music = "music/main.mp3";
 
     // Buttons
     Button start_game_button = Button();
     start_game_button.texture_path = "menu/button.png";
     start_game_button.alpha = 255;
     start_game_button.x = 960;
-    start_game_button.y = 340;
+    start_game_button.y = 400;
     start_game_button.x_scale = 400;
     start_game_button.y_scale = 100;
     start_game_button.text.text = "Play Game!";
     start_game_button.text.font_size = 72;
     start_game_button.text.color = { 255, 255, 255, 255 };
     start_game_button.text.x = 960;
-    start_game_button.text.y = 335;
+    start_game_button.text.y = 395;
     start_game_button.text.x_scale = 400;
     start_game_button.text.y_scale = 100;
     start_game_button.on_click = [gm = &game_controller]() {
-        gm->startGame();
+        gm->loadLobby();
     };
     this->buttons.push_back(start_game_button);
 
+    Button exit_game_button = Button();
+    exit_game_button.texture_path = "menu/button.png";
+    exit_game_button.alpha = 255;
+    exit_game_button.x = 960;
+    exit_game_button.y = 520;
+    exit_game_button.x_scale = 400;
+    exit_game_button.y_scale = 100;
+    exit_game_button.text.text = "Exit game";
+    exit_game_button.text.font_size = 72;
+    exit_game_button.text.color = { 255, 255, 255, 255 };
+    exit_game_button.text.x = 960;
+    exit_game_button.text.y = 515;
+    exit_game_button.text.x_scale = 400;
+    exit_game_button.text.y_scale = 100;
+    exit_game_button.on_click = [gm = &game_controller]() {
+        gm->exitGame();
+    };
+    this->buttons.push_back(exit_game_button);
+
     // Images
     Image logo = Image();
     logo.texture_path = "menu/logo.png";
     logo.alpha = 255;
     logo.x = 960;
-    logo.y = 210;
+    logo.y = 140;
     logo.x_scale = 680;
     logo.y_scale = 106;
     this->images.push_back(logo);
 }
 
 void MainMenu::start() {
+    // Remove remaining player components
+    auto& em = factory.getEntityManager();
+    auto player_entities = em.getEntitiesByComponent<PlayerComponent>();
+
+    for (auto& [ entity_id, player ] : player_entities ) {
+        em.removeEntity(entity_id);
+    }
```

Comment:
@Peter-JanGootzen Wasn't it your intention to have the players as a seperate re-usable *thing*, which I think is removed in this PR. Not that it is bad in the way it done, but it would become troublesome when saving the statistics of a user (@Bram-Boris ). As every game, they will be reset when a new stats component is created. I'd love to hear what your thoughts are on this Luuk and read in the stand up nodes how you are thinking about saving user's names.